src/rebar_prv_cover.erl

```diff
@@ -349,6 +349,8 @@ is_ignored(Dir, File, ExclMods) ->
 
 cover_compile_file(FileName) ->
     case catch(cover:compile_beam(FileName)) of
+        {'EXIT', Reason} ->
+            ?WARN("Cover compilation crashed: ~p", [Reason]);
```

Comment:
I assume this one bit here is to avoid a fatal core crash and just show the related issue? Any example of what this would look like?